src/Microsoft.AspNetCore.Authentication/Events/AuthenticationEvents.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    /// <summary>
+    /// Base class for all authentication events.
+    /// </summary>
+    public class AuthenticationEvents
```

Comment:
Using this as a base class isn't helpful when everybody just overrides it (and awkwardly at that).